### PR TITLE
nixos/base16-schemes unstable-2022-12-16 -> unstable-2023-05-02

### PR DESCRIPTION
**Pull request by TotalChaos05**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/themes/base16-schemes/default.nix [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "base16-schemes";
-  version = "unstable-2022-12-16";
+  version = "unstable-2023-5-2";
```
Comment (DamienCassou) [inline pkgs/data/themes/base16-schemes/default.nix:5]: I suggest to pad month and day with a 0 so the version string can be lexicographically sorted while keeping the date in the right order:

```
"unstable-2023-05-02"
```

**Review by DamienCassou — CHANGES_REQUESTED** [review]
thank you for your work. I left a suggestion

**Review by apfelkuchen6 — CHANGES_REQUESTED** [review]
Can you please squash all your commits into single one named "base16-schemes: unstable-2022-12-16 -> unstable-2023-05-02" and also change the PR title to that?

**Comment by TotalChaos05** [comment]
@apfelkuchen6 how would i combine the commits?

**Comment by DamienCassou** [comment]
@TotalChaos05: Search for "git squash" on the web, you will find different solutions among which:

1. either use git reset to delete all your commits but keep your changes and then make 1 commit
2. or use git rebase interactively and ask git to "squash" commits

I can do it for you if you don't want to.

**Comment by TotalChaos05** [comment]
@DamienCassou did that work?

**Comment by DamienCassou** [comment]
@TotalChaos05: everything went well, good job. The only remaining problem is the missing colon `:` after the package name in the commit message. See https://nixos.org/manual/nixpkgs/stable/#submitting-changes-making-patches.

File: pkgs/data/themes/base16-schemes/default.nix (hunk 1 above)
Comment (DamienCassou) [reply pkgs/data/themes/base16-schemes/default.nix:5]: It seems like the 0s have disappeared from your last commit.

**Review by DamienCassou — CHANGES_REQUESTED** [review]
The version number lacks the two 0s on the month and day.

**Comment by TotalChaos05** [comment]
sorry about all that :')

**Comment by DamienCassou** [comment]
Result of `nixpkgs-review pr 238995` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>base16-schemes</li>
  </ul>
</details>

**Comment by DamienCassou** [comment]
@TotalChaos05: There is nothing to be sorry about. Thank you for your work. Your next PR will be much easier to get merged! :-)

**Pull request merged**